src/neptune/internal/utils/hashing.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = ["generate_hash"]
+
+import hashlib
+
+
+def generate_hash(*descriptors, length: int = 8):
```

Comment:
I'm not a fan of such default value. Probably should be explicitly set to 8.

Reply:
Done